app/routes/_index.tsx

```diff
@@ -35,19 +53,31 @@ export default function Index() {
             typografi={TypografiTyper.StegHeadingH1}
           />
           <Språkvelger />
+
           <VeilederHilsen
             innhold={tekster.veilederhilsenInnhold}
             hilsen={tekster.brukerHilsen}
           />
           <SamtykkePanel
+            tittel={tekster.samtykkePanelTittel}
             innhold={tekster.samtykkePanelMelding}
             samtykke={tekster.samtykkePanelSamtykke}
             feilmelding={tekster.samtykkePanelFeilmelding}
```

Comment:
Her er det nok like greit å sende inn hele `tekster` objektet også destrukturere det inne i komponenten. Spør meg gjerne i morgen dersom dere lurer på hva jeg mener med destrukturering

Reply:
Tror jeg gjorde det nå, men ta gjerne en titt for å se om jeg gjorde det rett / sånn du tenkte!

Reply:
Ah kult! 🙌  For sanitynavn er jeg også litt usikker, men det vil jo ikke nødvendigvis tilhøre denne PR-en. Jeg kan opprette et Trello-kort for å undersøke det? 🕵️